tests/integration/components/collapsible-list-test.js

```diff
@@ -176,15 +177,56 @@ test('it should execute the onShow action when one is provided', function(assert
 });
 
 test('it should execute the onShow action when one is provided and animation is set to false', function(assert) {
-  assert.expect(1);
+  assert.expect(2);
 
-  this.set('doSomethingOnShow', () => {
+  this.set('doSomethingOnShow', (item) => {
     assert.ok(true);
+    assert.equal(item.name, 'item1');
   });
 
   this.render(hbs`
     {{#collapsible-list animation=false onShow=(action doSomethingOnShow) as |collapsible|}}
-      {{#collapsible.item as |item|}}
+      {{#collapsible.item name="item1" as |item|}}
+        {{#item.header}}header here...{{/item.header}}
+        {{#item.panel}}panel here...{{/item.panel}}
+      {{/collapsible.item}}
+    {{/collapsible-list}}
+  `);
+
+  click(SELECTORS.header);
+});
+
+test('it should execute the onAfterShow action when one is provided', function(assert) {
+  assert.expect(2);
+
```

Comment:
I think you probably want to do: `const done = assert.async();` here

Reply:
Makes sense, done.

Reply:
If the animation is false we don't need to make this asynchronous, the test is indeed green. Let me know what you think.